packages/garbo/src/fights.ts

```diff
@@ -438,6 +438,20 @@ function familiarSpec(underwater: boolean, fight: string): OutfitSpec {
       if (have($familiar`Reanimated Reanimator`)) {
         return { familiar: $familiar`Reanimated Reanimator` };
       }
+      if (
+        $monsters`Witchess Knight, Witchess Bishop`.includes(
```

Comment:
I'd really like to see something where we check the target's item drops to detect if goose is appropriate instead of hard coding the monster list in several places. Could you at the moment move this to a function so it's easier to update?

Reply:
Updated to a function; we'd need some large mafia changes to make that work. Specifically, we'd need drops to be presented in drop order in addition to whether they're duplicatable to make this truly generic.